packages/fxa-settings/src/pages/ResetPasswordRedesign/ConfirmResetPassword/index.tsx

```diff
@@ -0,0 +1,93 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import AppLayout from '../../../components/AppLayout';
+import FormVerifyTotp from '../../../components/FormVerifyTotp';
+import { ConfirmResetPasswordProps } from './interfaces';
+import { RouteComponentProps } from '@reach/router';
+import { useFtlMsgResolver } from '../../../models';
+import LinkRememberPassword from '../../../components/LinkRememberPassword';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { ResendEmailSuccessBanner } from '../../../components/Banner';
+import { ResendStatus } from '../../../lib/types';
+import { EmailCodeImage } from '../../../components/images';
+
+const ConfirmResetPassword = ({
+  email,
+  errorMessage,
+  resendCode,
+  resendStatus,
+  searchParams,
+  setErrorMessage,
+  setResendStatus,
+  verifyCode,
+}: ConfirmResetPasswordProps & RouteComponentProps) => {
+  const ftlMsgResolver = useFtlMsgResolver();
+
+  const localizedInputGroupLabel = ftlMsgResolver.getMsg(
+    'confirm-reset-password-otp-input-group-label',
+    'Enter 8-digit code'
+  );
+  const localizedSubmitButtonText = ftlMsgResolver.getMsg(
+    'confirm-reset-password-otp-submit-button',
+    'Continue'
+  );
+
+  const spanElement = <span className="font-bold">{email}</span>;
+
+  const handleResend = async () => {
+    const result = await resendCode();
+    if (result === true) {
+      setResendStatus(ResendStatus.sent);
+    }
+  };
+
+  return (
+    <AppLayout>
+      <FtlMsg id="confirm-reset-password-otp-flow-name">
+        <p className="text-start text-grey-400 text-sm">Reset password</p>
```

Comment:
This looks off. Just double checking that the CSS is as intended here.

Reply:
This matches the Figma design. I went back and forth on marking it as a heading considering the small font.

![image](https://github.com/mozilla/fxa/assets/22231637/670d416f-1b14-4200-8d87-098160dab40c)

Reply:
Sounds good. Just double checking!